docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>Let's put a link to nested-pandas docs <a href="https://nested-pandas.readthedocs.io/en/latest/" target="_blank">https://nested-pandas.readthedocs.io/en/latest/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Great idea! Thanks.</p>